### ISSUE TITLE
Godot Docs - Your First Game - GDScript code in text description

### ISSUE THREAD

**Issue by Esharsid** [godotengine/godot#20989]
**Godot version:**
 v3.0.6.stable.mono.official.8314054

**OS/device including version:**
Operating system: 
 Ubuntu 18.04 running under 
  VMWare Workstation 12 Player running on
   Windows 10 Pro

 Intel Core i7, 64-bit, x64

**Issue description:**
Following through the steps on "Your First Game" there were a couple times when there were references to GDScript in the text of the description that threw me while I was trying to follow along with the C# code.

**Steps to reproduce:**

(A) "extends Area2d"
http://docs.godotengine.org/en/3.0/getting_started/step_by_step/your_first_game.html#preparing-for-collisions
When coding in C#, the code snippet needs to be placed after "public class Player : Area2D
{".
![a-preparing for collisions](https://user-images.githubusercontent.com/42361511/44065833-fc0ee1c2-9f3a-11e8-8aab-3e329a491f87.JPG)

(B) “game_over” and "new_game"
http://docs.godotengine.org/en/3.0/getting_started/step_by_step/your_first_game.html#main-script
When coding in C#, the procedures are called "GameOver" and "NewGame".
![b-main script](https://user-images.githubusercontent.com/42361511/44065847-1525925a-9f3b-11e8-85ec-1232b1403642.JPG)

**Comment by akien-mga** [comment]
Such issue should be opened on the docs repository: https://github.com/godotengine/godot-docs

**Comment by aaronfranke** [comment]
This is intended behavior. One of the drawbacks of supporting multiple scripting languages with different casing schemes and only having one set of docs for it is that there will be minor inconsistencies. Users are expected to know that the code may be organized slightly differently and that anything written in snake_case in the docs is the PascalCase equivalent in C#.

**Comment by Esharsid** [comment]
Perhaps that disclaimer could be placed as a note in the documentation to warn the reader.

There aren't very many occurrences of code in the text. Those were the only two I saw, so since the documentation seemed really quite well done, I thought I'd point out where I had tripped-up.
The "extends Area2d" really did throw me for at least half an hour... I thought perhaps the C# script template was incomplete, or that I was looking at the wrong script (there is only one at that point in the build)... and frankly I just about gave up in frustration. Just thought a couple of words in the document might help smooth the path for others.

I turns out there's a third occurrence in the "Sound Effects" section
http://docs.godotengine.org/en/3.0/getting_started/step_by_step/your_first_game.html#sound-effects
but that one is actual code and could be handled with one of those selection tabs.


![capture](https://user-images.githubusercontent.com/42361511/44130730-df4e932e-a01c-11e8-9e75-df47e799ebda.JPG)

$Music.play()
		var music =(AudioStreamPlayer) GetNode("Music");
		music.Play();
$Music.stop()
		var music =(AudioStreamPlayer) GetNode("Music");
		music.Stop();

$DeathSound.play()
		var deathSound = (AudioStreamPlayer) GetNode("DeathSound");
		deathSound.Play();


Sorry about adding the issue(s) to the wrong repository.
